Lorax.py

```diff
@@ -171,12 +197,31 @@ def compareClades(l1, l2):
     outputFile.write(fileName)
     for clade, cladeIndexes in cladesOfInterestIndexes.items():
         confidenceLevel = "N/A" #defualt value, if we do not find the clade in the split
+
         for mx in matrixValues:
             bootstrapValue = mx[0]
             speciesIndexList = mx[1]
-            if compareClades(speciesIndexList, cladeIndexes):
+            if compareClades(speciesIndexList, cladeIndexes): 
                 confidenceLevel = str(bootstrapValue)
+                break
+
+        if len(cladeIndexes) < 2: #make sure there are at least 2 species in the clade seen in the loci
```

Comment:
for each clade, we create a list of all the species indexes we find in the split file.
if there are less then 2 specieses for that clade found in the split file, we do not want to consider it.